src/data/_contributors.yaml

```diff
@@ -1546,3 +1546,14 @@ maciejzasada:
     - author
   google: "104812516714503133849"
   email: maciej@unit9.com
+
+kennith:
```

Comment:
contributors file should be sorted by last name

Reply:
@petele fixed.